src/applications/personalization/dashboard/components/debts-v2/DebtsV2.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import DashboardWidgetWrapper from '../DashboardWidgetWrapper';
+import IconCTALink from '../IconCTALink';
+import recordEvent from '~/platform/monitoring/record-event';
+import {
+  fetchDebts,
+  fetchCopays,
+} from '~/applications/personalization/dashboard/actions/debts';
+import DebtsCardV2 from './DebtsCardV2';
+import CopaysCardV2 from './CopaysCardV2';
+
+const NoOutstandingDebtsText = () => {
+  return (
+    <p
+      className="vads-u-margin-bottom--2p5 vads-u-margin-top--0"
+      data-testid="no-outstanding-debts-text"
+    >
+      You have no outstanding debts or copays.
+    </p>
+  );
+};
+
+const OutstandingDebtsError = () => {
+  return (
+    <div className="vads-u-margin-bottom--2p5">
+      <va-alert
+        status="warning"
+        show-icon
+        data-testid="outstanding-debts-error"
+      >
+        <div className="vads-u-margin-top--0">
+          We’re sorry. We can’t access some of your financial information right
+          now. We’re working to fix this problem. Please check back later.
+        </div>
+      </va-alert>
+    </div>
+  );
+};

```

Comment:
Looks like this code is is basically the same within Dashboard, Debts, and DebtsV2. Maybe we can make this its own component in the future to reduce the repeated code.

Reply:
The code in Dashboard and Debts will go away during cleanup as that's from V1 but still a good point. I do like them in their own components, but really it will only be used in DebtsV2 after the flipper is removed